### DISCUSSION TITLE
Hi im a newbie to github

### DISCUSSION THREAD

**Discussion by Edblue3** [community/community#46217] [New to GitHub]
Hi I'm a newbie to github I would like to know how to code and also which software  I need to download to code

**Comment by jge162** [comment]
You can download vscode, pycharm. Both allow you to use copilot. And then go online and watch YouTube videos to program in the language of your choice. Good luck.

**Comment by DanteHaxor** [comment]
vscode for sure

**Comment by Ntshangase** [comment]
I think you need to start learning how to code using online sources like courser/YouTube and the downloading of software will be a natural progression from there.

**Comment by Juanescacha** [comment]
Just download VSCode and Git to your PC, learn with a youtube tutorial to how to use Git with Github in VSCode and you are ready to go!
then you can start learning any language you want or start with the basics of web like html, css, js and start building your projects! Good Luck

**Comment by ThefriendlyNeighbourhoodCoder** [comment]
Simply as an beginner go with vs code and with some of its extensions. :)

**Comment by Neodevils** [comment]
Visual Studio Code and Git would be enough foe you. Happy developing!
